Mailbox/Mailbox.cs

```diff
@@ -1,4 +1,38 @@
-﻿namespace Mailbox
+﻿using System;
+
+namespace Mailbox
 {
 
+    public class Mailbox
+    {
+        public Size _Size { get; set; }
+        public (int Row, int Column) _Location { get; set; }
+        public Person _Owner { get; set; }
```

Comment:
```suggestion
        public Person Owner { get; set; }
```

Believe these are just supposed to be regular camel case, I made the same mistake before Kevin set me straight.

As I understand it Properties should be normal camel case and fields should use an underscore. [Here's a link](https://stackoverflow.com/questions/295104/what-is-the-difference-between-a-field-and-a-property) to someone explaining the difference much better than I could.